fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/RollCall.java

```diff
@@ -185,18 +158,27 @@ public static String generateCloseRollCallId(String laoId, String closes, long c
     return Hash.hash(EventType.ROLL_CALL.getSuffix(), laoId, closes, Long.toString(closedAt));
   }
 
+  public static RollCall openRollCall(RollCall rollCall) {
+    return setRollCallState(rollCall, EventState.OPENED);
+  }
+
+  public static RollCall closeRollCall(RollCall rollCall) {
+    return setRollCallState(rollCall, EventState.CLOSED);
+  }
+
+  private static RollCall setRollCallState(RollCall rollCall, EventState state) {
+    RollCall updatedRollCall = new RollCall(rollCall);
+    updatedRollCall.state = state;
+    return updatedRollCall;
```

Comment:
To have a completely Immutable RollCall we need to mark the state as final. To do that, you could change this very small part to a second constructor (or builder call, as you think fits the best)

Something like
```suggestion
    return new RollCallBuilder(rollCall).setState(state).build;
```

Or something similar.